docs/nodejs.md

```diff
@@ -27,17 +27,6 @@ aa("init", {
 });
 ```
 
-## 3. Add `userToken`
```

Comment:
@JasonBerry do you know if this holds true? I created a tiny Node.js app that simply sends an Insights event and setting `userToken` on the event object didn't seem necessary. Let me know if you can think of a reason why we might require this

Reply:
Did the event validate once it hit Algolia or did it fail? I would expect it to fail. I believe we _want_ to remove the necessity of a user token being present on the backend for clicks but I don't think that work has been completed.

Reply:
As per Jason's Slack DM, there could be a good reason for this, e.g., anon ID might be generated per request in the Node env, so have added the text back in for now to err on the safe side. Also updated it to mention `authenticatedUserToken`.